src/Resources/public/js/pimcore/configuration/configItemDataObject.js

```diff
@@ -425,6 +426,10 @@ pimcore.plugin.storeExporterDataObject.configuration.configItemDataObject =
       });
       return this.accessForm;
     },
+    buildLoggerTab: function() {
+      const loggerTab = new pimcore.plugin.pimcoreDataImporterBundle.configuration.components.logTab(this.configName);
```

Comment:
im getting an error here, I think its because our test environment doesnt have the importer bundle. can you add this package to the composer.json or copy the relevant component into a helper file in this bundle.